Python/Product/Analysis/Interpreter/IPythonInterpreter.cs

```diff
@@ -59,11 +61,17 @@ public interface IPythonInterpreter : IDisposable {
         event EventHandler ModuleNamesChanged;
 
         /// <summary>
-        /// Returns an IPythonModule for a given module name. Returns null if
-        /// the module does not exist.
+        /// Synchronous variant of <see ref="ImportModuleAsync">.
+        /// Waits for the async import completion using timeouts.
         /// </summary>
         IPythonModule ImportModule(string name);
 
+         /// <summary>
+        /// Returns an IPythonModule for a given module name. Returns null if
+        /// the module does not exist. The import is performed asynchronously.
+        /// </summary>
+        Task<IPythonModule> ImportModuleAsync(string name, CancellationToken token);
```

Comment:
This needs to go onto a separate interface.

Reply:
`IPythonInterpreter2` ?